src/Behat/Behat/Context/Environment/InitializedContextEnvironment.php

```diff
@@ -24,6 +24,8 @@
  * @see ContextEnvironmentHandler
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * @template T of Context
```

Comment:
We define T globally for the whole class as we are going to use it in a couple of places